### PR TITLE
[Security] update @unvt/sprite-one v0.0.9 to v0.1.1

### PR DESCRIPTION
**Pull request by yuiseki**
## TL;DR

- This Pull Request updates `@unvt/sprite-one` to v0.1.1.
- `@unvt/sprite-one` depends on `sharp`, which only officially supports Node.js v18 and above
  - Therefore, we need to make a decision on whether `@unvt/charites` will continue to support Node.js v14, v16
- This Pull Request proposes to remove Node.js v14, v16 and add v20, v21 from `.github/workflows/build.yml`

## Description

- `@unvt/charites` includes `@unvt/sprite-one` in dependencies.
- `@unvt/sprite-one` v0.0.9 includes `sharp` in dependencies.
- `sharp` includes `libwebp` in dependencies.
- It has been discovered that `libwebp` is vulnerable to [CVE-2023-4863](https://access.redhat.com/security/cve/cve-2023-4863).
Fortunately, this vulnerability has already been fixed.
- We can prevent this vulnerability by setting `@unvt/sprite-one` to v0.1.1.

So,
- This Pull Request updates `@unvt/sprite-one` to v0.1.1.

## Type of Pull Request

- [ ] Adding a feature
- [ ] Fixing a bug
- [ ] Maintaining documents
- [x] Others (Fixing security issue)

## Verify the followings

- [x] Code is up-to-date with the `main` branch
- [x] No build errors after `npm run build`
- [x] No lint errors after `npm run lint`
- [x] No errors on using `charites help` globally
- [ ] Make sure all the existing features working well
- [ ] Have you added at least one unit test if you are going to add new feature?
- [x] Have you updated documentation?

Refer to [CONTRIBUTING.MD](https://github.com/unvt/charites/tree/master/.github/CONTRIBUTING.md) for more details.

### CODE REVIEW

**Comment by keichan34** [comment]
I think we can drop support for nodejs 16 and under, since they're already out of the [LTS maintenance period](https://nodejs.org/en/about/previous-releases). We should probably add 20.x to the test matrix as well

**Review by keichan34 — APPROVED** [review]

**Review by hfu — APPROVED** [review]

**Review by smellman — CHANGES_REQUESTED** [review]
https://github.com/unvt/charites/blob/2cd6f0259a6498cc42511db0f158d6a968feff2d/docs/source/install/install.rst#L3

You need update document

**Comment by yuiseki** [comment]
@smellman Thank you for your suggestion! I have updated the documentation. Please re-review 🙏

**Review by smellman — APPROVED** [review]
LGTM!
